.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,9 @@
-Thanks for contributing code! Please include a description of your change and
-check your PR against the list below (for further questions, refer to the
-[contributing guide](https://github.com/Cantera/cantera/blob/master/CONTRIBUTING.md)).
+Thanks for contributing code! Please include a description of your change and check your PR against the list below (for further questions, refer to the [contributing guide](https://github.com/Cantera/cantera/blob/master/CONTRIBUTING.md)).
```

Comment:
I agree with the suggestion to move this into an HTML comment. In addition, a small wording suggestion:
```
check your PR against the list below. For further questions...
```
i.e., turn the parenthetical statement into a full sentence.